pkg/apis/build/v1alpha1/build.go

```diff
@@ -89,7 +89,7 @@ func (b *Build) Finished() bool {
 	return !b.Status.GetCondition(duckv1alpha1.ConditionSucceeded).IsUnknown()
 }
 
-func (b *Build) BuildEnvVars() []corev1.EnvVar {
+func (b *Build) SourceEnvVars() []corev1.EnvVar {
```

Comment:
This name is a little confusing, but I understand why you want to change it. Unfortunately, I don't have a suggestion to make.

Reply:
BuildtimeEnvVariables

Reply:
I changed this because working on this kept confusing me. 
It is not really a Build env var because it is just the input to setup of the configured build source.

Reply:
BuildtimeEnvVariables would imply it is the build.spec.env?

Reply:
But this is what these variables are correct?

Reply:
No, These are the env vars for app source. 

Like GIT_URL & GIT_REVISON.

Reply:
Oh in that case this change makes sense. I thought these were build time env vars as well.